public/projects/international-accessibility-task-force/iatf-bot.mdx

```diff
@@ -0,0 +1,141 @@
+---
+name: International Accessibility Task Force [discord bot]
+repoUrl: https://github.com/international-accessibility-task-force/iatf-bot
+description: International Accessibility Task Force is an open-to-everyone community of assistive technology users, open-source developers, and people interested in accessibility and inclusion.
+languages: # A list of programming languages used in this project
+  - JavaScript
+  - NodeJS
+  - DiscordJS
+  - HTML / CSS
+  - Markdown
+tags: # A list of keywords relevant to the contributors for this project
+  - First Timer Friendly
+  - Social Activism
+  - Accessibility
+  - Community
+  - Open-Source
+  - Discord
+  - Bot
+  - npm
+  - lint
+  - eslint
+  - babel
+  - a11y
+avatar: logo-dearalgorithm.png
+websiteUrl: https://internationalaccessibilitytaskforce.com/
+currentlySeeking: # (optional) These fields are searchable and very helpful for contributors to match well with your project
+  - Developers
+  - Backend Developers
+  - Frontend Developers
+  - DevOps
+  - Designers
+  - Maintainers
+  - Code Reviewers
+  - Technical Writers
+  - Mentors
+  - Testers
+  - UX
+contributionOverview: # (optional) Provide additional context to help contributors match well with your project
+  mainLocation: Europe
+  idealEffort: 1 PR a month
+  isMentorshipAvailable: true
+  automatedDevEnvironment: https://www.github.io/
```

Comment:
**suggestion:** @mugiwarafx could you please verify that this information is correct?

Reply:
Hi @deammer, yes, it is. Maybe Github Pages & Github Actions would be more specific for the last point, but I thought that saying Github was enough since englobes both. 🙂